src/app/pages/test/Test.js

```diff
@@ -1,4 +1,4 @@
-import { TextField, RadiosField } from 'common/forms'
+import { TextField, RadiosField } from '../../common/forms'
```

Comment:
this path should be relative to project folder. so it was correct before.
it seems in jest this can be handled by `modulePaths` option